packages/workbox-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -219,9 +330,7 @@ class RequestWrapper {
 
     if (this.plugins.has('requestWillFetch')) {
       for (let plugin of this.plugins.get('requestWillFetch')) {
-        const returnedPromise = plugin.requestWillFetch({request});
-        isInstance({returnedPromise}, Promise);
-        const returnedRequest = await returnedPromise;
+        const returnedRequest = await plugin.requestWillFetch({request});
```

Comment:
This logic is equivalent to what was there before; because I'm now using `await`, the return value from `plugin.requestWillFetch()` will always be a promise.